doc/development/powershell.md

```diff
@@ -0,0 +1,179 @@
+This page contains guidelines for developing or updating powershell scripts used by the Azure SDK engineering system.
+
+Table of Contents
+=================
+
+* [TLDR](#tldr)
+* [Structure](#structure)
+* [Functionality](#functionality)
```

Comment:
Putting in a reminder for myself here to call out NOT calling functions via `myFunc(1, 2)`.